src/wpool_queue_manager.erl

```diff
@@ -288,8 +336,9 @@ handle_call(
       case erlang:is_process_alive(ClientPid) andalso
            Expires > now_in_microseconds() of
         true  ->
+          NewState = monitor_worker(Worker, Client, State#state{workers = NewWorkers}),
```

Comment:
Line 339 is too long:           NewState = monitor_worker(Worker, Client, State#state{workers = NewWorkers}),.